scipy/sparse/_compressed.py

```diff
@@ -658,6 +686,42 @@ def _minor_reduce(self, ufunc, data=None):
     # Getting and Setting #
     #######################
 
+    def _get_int(self, idx):
+        if 0 <= idx <= self.shape[0]:
+            spot = np.flatnonzero(self.indices == idx)
+            if spot.size:
+                return self.data[spot[0]]
+            return self.data.dtype.type(0)
+        raise IndexError(f'index ({idx}) out of range')
+
+#    def _get_slice(self, idx):
```

Comment:
May be nice to generalize this eventually, so we don't need special case for 1d, 2d, etc.